src/hooks/useDrag.tsx

```diff
@@ -54,15 +59,20 @@ const updateEditorSection = (
   homepageState: EditorHomepageState,
   newDisplaySections: unknown[],
   newFrontMatterSection: EditorHomepageState["frontMatter"]["sections"],
-  newSectionErrors: unknown[]
+  newSectionErrors: unknown[],
+  newTextcardErrors: unknown[][]
 ): EditorHomepageState => ({
   ...homepageState,
   displaySections: newDisplaySections,
   frontMatter: {
     ...homepageState.frontMatter,
     sections: newFrontMatterSection,
   },
-  errors: { ...homepageState.errors, sections: newSectionErrors },
+  errors: {
+    ...homepageState.errors,
+    sections: newSectionErrors,
+    textcards: newTextcardErrors,
+  },
 })
 
```

Comment:
why do we need the extra argument? if `homepageState.errors.textcards` is already updated then the spread of `...homepageState.errors` should be sufficient, no?

Reply:
This call is made when a new section is created - `homepageState.errors.textcards` needs to be modified here because the length of the textcards errors should also match the length of sections